design.md

```diff
@@ -0,0 +1,43 @@
+# Epiverse search
+
+## User persona & user story
+
+This work is mostly targeted at the following persona, with the following stories:
+
+- As a PEI tool implementer, I want to find out if there is an R package to do this specific epidemiological analysis task I have in mind. This allows me to quickly find the right tool for the job.
+- As a PEI tool creator, I want to find out if there is an R package to do this specific epidemiological analysis task I have in mind. This prevents me from reinventing the wheel and increasing the ecosystem fragmentation.
+
+These user stories have been collected at:
+
+- the '100 days and 100 lines of code' workshop (December 2022)
+- the 'Solution for Collaboratory' launch workshop (January 2023)
+
+## Requirements
+
+The search functionality should:
+
+- take natural language text description of the task the user is trying to perform (e.g., "I want to estimate the risk of a dengue outbreak in Colombia in the next 3 months")
+- return a (list of) valid R package(s) performing the described task, taken from the CRAN Task View in Epidemiology
```

Comment:
To be certain: It is out of scope to return the way said package performs the described task.

Is this correct?

Reply:
At this point, definitely!

The question of whether it should be on the roadmap for the future and we should design with this in mind is still unresolved I'd say.